src/coordinate_transformations.jl

```diff
@@ -0,0 +1,91 @@
+mutable struct CRS2CRS <: CoordinateTransformations.Transformation
+    rep::Ptr{Cvoid}
+    direction::PJ_DIRECTION
+end
+
+function CRS2CRS(source::String, target::String, direction::PJ_DIRECTION = PJ_FWD; area = C_NULL, normalize = true)
+
+    if direction == PJ_IDENT || source == direction
+        return IdentityTransformation()
+    end
+
+    pj_init = proj_create_crs_to_crs(source, target, area)
+
+    obj = if normalize
+        pj_normalized = proj_normalize_for_visualization(pj_init)
+        proj_destroy(pj_init)
+        CRS2CRS(pj_normalized, direction)
+    else
+        CRS2CRS(pj_normalized, direction)
+    end
+
+    finalizer(obj) do obj
+        proj_destroy(obj.rep)
+    end
+
+    return obj
+end
+
+function CRS2CRS(source::Projection, target::Projection, direction::PJ_DIRECTION = PJ_FWD; area = C_NULL, normalize = true)
+    return CRS2CRS(
+        sprint(print, source),
+        sprint(print, target),
```

Comment:
Rather than using `sprint` here, I see that there's `proj_create_crs_to_crs_from_pj` which might be useful?

Reply:
I tried that, but got pretty consistent segfaults.  Not sure why...